bedrock/base/templates/base-protocol.html

```diff
@@ -53,7 +53,7 @@
 
     <!--[if lt IE 9]>
       {# Only needed for IE before v9 #}
-      <script type="text/javascript" src="{{ static('js/ie/libs/html5shiv.js') }}"></script>
+      <script src="{{ static('js/ie/libs/html5shiv.js') }}"></script>
```

Comment:
If this script is for IE <= 9, do we need to keep the type attr? I'm not sure on the answer!

Reply:
Seems like no. It appears that it never did anything for the actual browser parsers 😝 (probably why it isn't necessary anymore)